query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/WfsVersion.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.sources.fields;
+
+import java.util.List;
+
+import org.codice.ddf.admin.api.fields.Field;
+import org.codice.ddf.admin.common.fields.base.BaseEnumField;
+import org.codice.ddf.admin.common.fields.base.scalar.StringField;
+
+import com.google.common.collect.ImmutableList;
+
+public class WfsVersion extends BaseEnumField<String> {
+
+    public static final String DEFAULT_FIELD_NAME = "wfsVersion";
+
+    public static final String TYPE_NAME = "WfsVersion";
+
+    public static final String DESCRIPTION = "The version of the WFS source.";
+
+    public static final String WFS_10 = "1.0.0";
```

Comment:
There is no service property for the wfs version. This value is used strictly for mapping to the factory pid.